From ef92f2343739d4a0ba21aafb3098040dfab6e27b Mon Sep 17 00:00:00 2001
From: martell <martellmalone@gmail.com>
Date: Fri, 12 Sep 2014 01:06:15 +0100
Subject: [PATCH 5/6] Change both fs case-sensitivity Aborts to Log Warns

As otherwise Windows cannot be used as the build machine.
---
 scripts/crosstool-NG.sh.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/crosstool-NG.sh.in b/scripts/crosstool-NG.sh.in
index 53ac552..f5caf2d 100644
--- a/scripts/crosstool-NG.sh.in
+++ b/scripts/crosstool-NG.sh.in
@@ -90,7 +90,7 @@ CT_DoExecLog DEBUG rm -f "${CT_WORK_DIR}/backtrace"
 
 # Check build file system case-sensitiveness
 CT_DoExecLog DEBUG touch "${CT_WORK_DIR}/foo"
-CT_TestAndAbort "Your file system in '${CT_WORK_DIR}' is *not* case-sensitive!" -f "${CT_WORK_DIR}/FOO"
+CT_DoLog WARN "Your file system in '${CT_WORK_DIR}' is *not* case-sensitive!" -f "${CT_WORK_DIR}/FOO"
 CT_DoExecLog DEBUG rm -f "${CT_WORK_DIR}/foo"
 
 # Check the user is using an existing SHELL to be used by ./configure and Makefiles
@@ -272,7 +272,7 @@ CT_DoExecLog ALL mkdir -p "${CT_HOST_COMPLIBS_DIR}"
 
 # Check install file system case-sensitiveness
 CT_DoExecLog DEBUG touch "${CT_PREFIX_DIR}/foo"
-CT_TestAndAbort "Your file system in '${CT_PREFIX_DIR}' is *not* case-sensitive!" -f "${CT_PREFIX_DIR}/FOO"
+CT_DoLog WARN "Your file system in '${CT_PREFIX_DIR}' is *not* case-sensitive!" -f "${CT_PREFIX_DIR}/FOO"
 CT_DoExecLog DEBUG rm -f "${CT_PREFIX_DIR}/foo"
 
 # Kludge: CT_INSTALL_DIR and CT_PREFIX_DIR might have grown read-only if
-- 
2.2.1

